packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Interaction Code grant type
+meta:
+  - name: description
+    content: An overview of the Interaction Code grant type for Okta Identity Engine.
+---
+
+# Interaction Code grant type
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+## Overview
+
+To enable a more customized user authentication experience, Okta has introduced an addition to the [OAuth 2.0 and OpenID Connect](/docs/concepts/oauth-openid) standard called the Interaction Code grant type. This grant type allows apps to manage user interaction with Okta without a browser. This is useful when the client has a particular way that it wants to interact with the user and doesn’t need to share an authenticated session with other applications.
+
+The Interaction Code flow consists of a series of interactions between the user and an Okta org, facilitated by the client. Each interaction is called a remediation step and corresponds to a piece of user data required by the Okta org for authentication. The client obtains these remediation steps from the Identity Engine component of the Okta org and prompts the user for the required data to continue the flow. Each step of the remediation is a criteria of assurance for authenticating the user, and is evaluated at runtime based on predefined Okta org and application sign-on policies and rules. The number and nature of remediation steps are configurable in the policies with minimal disruption to the running application. This allows the authentication experience to differ based on user, group, context, application, available factors, and so on.
```

Comment:
"prompts the user for the required data to continue the flow." -> "prompts the user for the required data to continue the flow at each step." ? Or "at each stage", maybe ?